index.html

```diff
@@ -1481,8 +1486,8 @@ <h3><a aria-current="false" data-automation-id="jobTitle"
                                         <div data-automation-id="jobSidebar" class="css-9bbna8">
                                             <div data-automation-id="sidebar" class="css-1gw0f0n">
                                                 <div data-automation-id="imageSection" class="css-6qub7p">
-                                                    <h3 id="11" class="css-1uppmgb">About Us</h3>
-                                                    <img src="./assets/logo.png" alt="Logo" data-automation-id="image" class="css-1mh151z">
+                                                    <h3 id="11" class="css-1uppmgb">About Us</h2>
+                                                    <img src="./assets/logo.png" alt="silhouette of the Boulder Flatirons" data-automation-id="image" class="css-1mh151z">
```

Comment:
This image did not have a properly descriptive alt tag, which it should have since it is descriptive content. The case could be made, though, for this being decorative, which would still require a change of alt text to "".